x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/PositionToXContent.java

```diff
@@ -166,6 +166,14 @@ protected XContentBuilder valueToXContent(XContentBuilder builder, ToXContent.Pa
                     }
                 }
             };
+            case "float" -> new PositionToXContent(block) {
+                @Override
+                protected XContentBuilder valueToXContent(XContentBuilder builder, ToXContent.Params params, int valueIndex)
+                    throws IOException {
+                    Float val = Float.intBitsToFloat(((IntBlock) block).getInt(valueIndex));
```

Comment:
in "our" case I think it is, but I wonder if the block in general might be of other concrete type and hence throw CCE here.

Reply:
we'll probably add a FloatBlock at some point, but for now IntBlock is fine.